content/intro/introduction.tex

```diff
@@ -1,39 +1,63 @@
+The ordering of the dates drawn and the birthdays of all men then determined the order of enrollment.
+The lottery was quickly proven to be botched; statisticians noticed an anomalous pattern.
+The broadcast was reviewed, and the drum was always turned an even number of times, such that capsules on top stayed on top.
+In other words, there was not sufficient mixing to make it a statistically random draw~\cite{princeton2016}.
+
+It is not only governments or large organizations, that can bias randomness.
+For example, a single adversary might be able to affect a seemingly random outcome, and thereby exploit the initial trust in an otherwise honest system.
+This happened in 2015 where the security director at Multi-State Lottery Association, successfully installed a rootkit on the trusted computer, used for generating the randomness for the lottery draw.
+He did it by exercising his privileges as security director, to temporarily disable the video surveillance, which otherwise was constantly filming the computer situated in a glass cage.
+The exploit allowed him to predict the next lottery numbers, and claim the prize of \$14.3m.
+It was later discovered that he had manipulated the system, after another video surveillance system, caught him buying the winning ticket~\cite{bbclotteryexploit}.
+
+% TODO: Give some example like lotto (and perhaps another)
+
+These examples show that it is difficult to generate public randomness and convince the public that it is truly random --- the process might not be sufficiently random, or the public might not believe it.
+
+% TODO: However, creating secure randomness is not easy. Give an example of the NIST beacon insecurity and unavailability.
+Even today with computers available, creating public randomness is not easy. Rabin introduced the term \emph{beacon}~\cite{rabin1983transaction}, a trusted service to provide fresh randomness at regular intervals. The idea is that nobody can predict the next value in advance.
+
+The \gls{nist} beacon is perhaps the most recognized randomness beacon available.
+It is run by the US Government, and broadcasts 512 random bits every minute.
+If this beacon could be trusted, it would be universally usable. The random bits are trivially converted into numbers, and are easily usable by national lotteries and even in your local bingo club every Wednesday night.
+However, there is no proof that the bits have been generated by the quantum machine as they claim. You have to trust that somewhere in a building in Maryland, the bits are generated as described.
+Their reputation is somewhat tarnished, as they were involved in the publication of the Dual Elliptic Curve PRNG standard, which they now have admitted contained a back door~\cite{nist2014backdoor}.
+Furthermore, a centralized beacon may be unreliable --- the \gls{nist} beacon was offline during the 2013 US Government shutdown~\cite{bonneau2015bitcoin}.
+
+\textcolor{orange}{TODO: Randomness is also important in cryptography.}
```

Comment:
Overlap med usecase-crypto - måske kun have den ene her i stedet for at dække alt i introen ? Den kan blive meget lang.